packages/integrations/stores/split/src/split.ts

```diff
@@ -0,0 +1,42 @@
+import type { EvaluationContext, Provider } from '@openfeature/server-sdk';
+import { OpenFeatureConfigStore } from '@configu/integrations/src/utils/OpenFeature';
+import { SplitFactory } from '@splitsoftware/splitio';
+import { OpenFeatureSplitProvider } from '@splitsoftware/openfeature-js-split-provider/src';
+import type { Attributes, IClient, SplitKey, Treatment, Treatments } from '@splitsoftware/splitio/types/splitio';
+
+export type SplitConfigStoreConfiguration = {
+  authKey: string;
+  key?: SplitKey;
+  context?: EvaluationContext;
+};
+
+export class SplitConfigStore extends OpenFeatureConfigStore {
+  private readonly splitClient: IClient;
+
+  constructor(configuration: SplitConfigStoreConfiguration) {
+    const { authKey, context } = configuration;
+    const splitClient = SplitFactory({ core: { authorizationKey: authKey } }).client();
+    const provider = new OpenFeatureSplitProvider({ splitClient }) as Provider;
+
+    super({
+      provider,
+      context,
+    });
+
+    this.splitClient = splitClient;
+  }
+
+  getTreatment = (featureFlag: string, attributes?: Attributes): Treatment => {
```

Comment:
can you please use a regular function instead of an arrow function?
it's confusing because you also this `this` as a reference to the class instance.

Reply:
Update